src/client/index.css

```diff
@@ -55,68 +73,120 @@ a {
   color: #2d3233;
 }
 
+.border {
+  box-shadow: 0px 0px 50px #eeeeee;
+  padding: 2em;
+}
+
 .btn {
-  background-color: #325288; 
+  background-color: #24b091;
   border: none;
   border-radius: 6px;
   color: white;
-  padding: 0.7rem 1.4rem;
+  padding: 1em 1.4em;
   text-align: center;
   text-decoration: none;
   display: inline-block;
-  font-size: 1rem;
+  font-size: 1.2rem;
   font-weight: 600;
   transition-duration: 0.5s;
-  width: fit-content;
+  width: 100%;
+}
+
+.btn:disabled {
+  background-color: lightgray;
 }
 
 .btn:hover {
   color: white;
-  background-color: #24A19C;
+  background-color: #fcc82f;
 }
 
 header {
-  background-color: #efefef;
-  margin-top: 2em;
+  margin: 2em 0em 1em 0em;
   display: flex;
   justify-content: space-between;
   align-items: center;
 }
 
 nav ul {
-display: flex;
-gap: 1.5em;
+  display: flex;
+  gap: 1.5em;
 }
 
 footer {
-  background-color: #efefef;
-  margin-top: 2em;
+  margin-top: 1em;
   display: flex;
   flex-direction: column;
-  margin: 3em 0 2em 0;
+  margin: 3em 0 0.7em 0;
+  gap: 5em;
 }
 
-.footerSectionWrapper {
+footer ul {
   display: flex;
-  flex-direction: row;
+  gap: 1em;
+}
+
+.footerSection {
+  display: flex;
+  flex-direction: column;
   align-items: center;
-  justify-content: space-between;
+  justify-content: center;
 }
 
-.footerSectionWrapper > div {
+.footerSection > div {
   display: flex;
   gap: 1.5em;
 }
 
+.footerSection:last-child {
+  font-size: 0.7em;
+}
 
-/* @media screen and (max-width: 1400px) {
-  #root {
+.socialMedia {
+  margin: 2em 0;
+  width: 2em;
+  filter: invert(48%) sepia(95%) saturate(312%) hue-rotate(117deg)
+    brightness(98%) contrast(98%);
+}
 
-  }
+.socialMedia:hover {
+  filter: invert(72%) sepia(6%) saturate(3120%) hue-rotate(293deg)
+    brightness(99%) contrast(94%);
 }
 
-@media screen and (max-width: 750px) {
-  #root {
+@media screen and (max-width: 1000px) {
+  header {
+    margin: 2em 0em 1em 0em;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: space-between;
+  }
+h1 {
```

Comment:
I strongly suggest you not to style the elements with their element names. instead fo ex: reviewHeading1, mealHeading2 etc.